source/SnapDump-Server/SnapDumpAPI.class.st

```diff
@@ -16,7 +16,7 @@ SnapDumpAPI class >> rootCallClass [
 SnapDumpAPI >> info [
 	^ OAInfo new 
 		description: 'SnapDump API';
-		version: '0.7';
+		version: SDServer version;
```

Comment:
This version is meant to be the API version not the server build version. If we change e.g. the filesystem layout the REST API would sty the same but we would raise the version number of the server of course. It would be then a version 3.x SnapDump distribution with a v2 REST API version

Reply:
ok that's right, let me update this PR as soon as the apiVerison of Cyrille is merged

Reply:
Good idea ;)